packages/callout/src/Callout.tsx

```diff
@@ -208,7 +205,20 @@ function Callout({
           className={headerIconStyle}
           role="presentation"
         />
-        {headerLabels[variant]}
+        <Overline
+          as="h3"
+          className={cx(
+            css`
+              color: inherit;
+              font-size: 12px;
+              font-weight: 500;
+              letter-spacing: 1px;
+              line-height: 16px;
+            `,
```

Comment:
Not sure if this text should have one-off styling that isn't based on the theme or other centralized values for font size, weight, etc., but keeping things as is for parity.

Reply:
The updated `Callout` design files use the same styles as Typography's `Overline` https://www.figma.com/file/4h2VwjCuJUbeZ7hzD2J1rq/LeafyGreen-Design-System?node-id=5821%3A44855

Reply:
This seems like a change that happened after Shaneeza finished the redesign

Reply:
It looks like the font-weight and letter-spacing are still different, so I've just kept those 2 lines now.

Reply:
I think font weight should be `700` still (we need to document this somewhere, but `Semibold` in Figma translates to bold/700 in CSS)